cfme/infrastructure/host.py

```diff
@@ -211,6 +206,11 @@ def power_off(self):
     def get_power_state(self):
         return self.get_detail("Properties", "Power State")
 
+    def provide(self):
+        """Provide Openstack infra node"""
+        view = navigate_to(self, 'Details')
+        view.toolbar.configuration.item_select('Provide Node', handle_alert=True)
```

Comment:
**REQUIRED**
Please do not add specific methods into common class. Inherit it and implement this method there.